controllers/ovs/flowsconfig.go

```diff
@@ -0,0 +1,144 @@
+package ovs
+
+import (
+	"context"
+	"fmt"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	flowsv1alpha1 "github.com/netobserv/network-observability-operator/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	ovsConfigName = "ovs-flows-config"
+)
```

Comment:
Any reason to not use constants.go file ?

Reply:
I think as long a constant isn't needed across several packages it's better to keep them in their package